### PR TITLE
 Optimized layout of PaiMode.md

### PR DESCRIPTION
**Pull request by 98may**
typeset pictures and add catalog in PaiMode.md

### CODE REVIEW

File: docs/en_US/TrainingService/PaiMode.md [hunk 1]
```diff
@@ -2,25 +2,27 @@
 ===
 NNI supports running an experiment on [OpenPAI](https://github.com/Microsoft/pai), called pai mode. Before starting to use NNI pai mode, you should have an account to access an [OpenPAI](https://github.com/Microsoft/pai) cluster. See [here](https://github.com/Microsoft/pai#how-to-deploy) if you don't have any OpenPAI account and want to deploy an OpenPAI cluster. In pai mode, your trial program will run in pai's container created by Docker.
```
Comment (4696047344) [inline docs/en_US/TrainingService/PaiMode.md:3]: Run

**Review by 4696047344 — APPROVED** [review]

File: docs/en_US/TrainingService/PaiMode.md [hunk 2]
```diff
@@ -190,4 +205,4 @@ Check policy:
 3. Note that the version check feature only check first two digits of version.For example, NNIManager v0.6.1 could use trialKeeper v0.6 or trialKeeper v0.6.2, but could not use trialKeeper v0.5.1 or trialKeeper v0.7.
 
 If you could not run your experiment and want to know if it is caused by version check, you could check your webUI, and there will be an error message about version check.
-![](../../img/version_check.png)
+![](../../img/version_check.png)
```
Comment (Lijiaoa) [inline docs/en_US/TrainingService/PaiMode.md:208]: Looks like this change should be revert. Please help check.
image

File: docs/en_US/TrainingService/PaiMode.md [hunk 3]
```diff
@@ -88,6 +93,8 @@ paiConfig:
 
 Note: You should set `trainingServicePlatform: pai` in NNI config YAML file if you want to start experiment in pai mode. The host field in configuration file is PAI's job submission page uri, like `10.10.5.1`, the default http protocol in NNI is `http`, if your PAI's cluster enabled https, please use the uri in `https://10.10.5.1` format.
 
+
+
 ### Trial configurations
 
 Compared with [LocalMode](LocalMode.md) and [RemoteMachineMode](RemoteMachineMode.md), `trial` configuration in pai mode have these additional keys:
```
Comment (Lijiaoa) [inline docs/en_US/TrainingService/PaiMode.md:100]: ->  `trial` configuration **on** pai mode **has** additional keys
I think `have these additional keys ` is too chinese english...
@QuanluZhang It's right?

Comment (QuanluZhang) [reply docs/en_US/TrainingService/PaiMode.md:100]: "have these additional keys" -> "has the following additional keys"

**Review by liuzhe-lz — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

**Pull request merged**